adam-core/src/main/java/org/bdgenomics/adam/io/FastqRecordReader.java

```diff
@@ -145,12 +145,23 @@ protected FastqRecordReader(final Configuration conf,
         CompressionCodecFactory codecFactory = new CompressionCodecFactory(conf);
         CompressionCodec codec = codecFactory.getCodec(file);
 
-        // we expect this record reader to be used only with input formats that
-        // extend FastqInputFormat. the behavior of this record reader depends on
-        // whether the input stream is splittable or not. FastqInputFormat maintains
-        // a contract where it will put the file's splittable status into the hadoop
-        // configuration object.
-        isSplittable = conf.getBoolean(FastqInputFormat.FILE_SPLITTABLE, false);
+	// if our codec is splittable, we can (tentatively) say that
```

Comment:
Thanks for catching this; I was editing this patch on a different computer from my usual and I was wondering why the diff looked weird.